ui/src/app/layout/dashboard/dashboard.component.ts

```diff
@@ -84,6 +92,47 @@ export class DashboardComponent implements OnInit {
         return moment(timestamp).locale(this.translate.currentLang).format('L LT');
     }
 
+    getInitialsProfilePicture(user: string): string {
+        const name = user.split('@')[0].replace('.','+');
+        return `https://ui-avatars.com/api/?name=${name}&background=random`;
+    }
+
+    getOutlookProfilePicture(user: string): string {
+        return `https://outlook.office.com/owa/service.svc/s/GetPersonaPhoto?email=${user}&size=HR64x64`;
+    }
+
+    getGravtarProfilePicture(user: string): string {
+        const md5 = new Md5();
+        md5.appendStr(user.trim().toLowerCase());
+        const hash = md5.end();
+        return `https://www.gravatar.com/avatar/${hash}?d=identicon`;
+    }
+
+    getProfilePicture(user: string) {
+        switch (this.changelogProfilePicture.toLowerCase()) {
+            case 'outlook':
+                return this.getOutlookProfilePicture(user);
+                break;
+            case 'gravtar':
+                return this.getGravtarProfilePicture(user);
+            case 'initials':
+                return this.getInitialsProfilePicture(user);
+            default:
+                return '/assets/images/default_avatar.png';
+        }
+    }
+
+    getDefaultPicture(user: string) {
+        switch (this.changelogDefaultPicture.toLowerCase()) {
```

Comment:
Code could be combined with getProfilePicture  (e.g. `getPicture(user: string, pictureType: string)` )